nntrainer/models/neuralnet.cpp

```diff
@@ -32,6 +32,7 @@
 #include <nntrainer_error.h>
 #include <nntrainer_log.h>
 #include <parse_util.h>
+#include <profiler.h>
```

Comment:
I think it would be better to enable this depending on profile options. No need include this without profile options

Reply:
Below is the code if the profiler is not enabled for `profiler.h`
```cpp
#ifndef PROFILE

#define START_PROFILE(event_key)
#define END_PROFILE(event_key)

#else
#define START_PROFILE(event_key)                         \
  do {                                                   \
    auto &prof = nntrainer::profile::Profiler::Global(); \
    prof.start(event_key);                               \
  } while (0);

#define END_PROFILE(event_key)                           \
  do {                                                   \
    auto &prof = nntrainer::profile::Profiler::Global(); \
    prof.end(event_key);                                 \
  } while (0);
#endif
```
This enables the use of `START_PROFILE` without #ifdef